test/integration/sign-in-test.js

```diff
@@ -85,3 +85,36 @@ test('sign in', function (t) {
 
   .catch(t.error)
 })
+
+test('sign in while signed in fails', function (t) {
```

Comment:
The test name seems a bit ambiguous. Wouldn't `prevent sign-in when already signed in` be better?